src/components/Navigation/navigation.css

```diff
@@ -0,0 +1,64 @@
+:root{
+    --sp-0-75:0.75rem;
+}
+
+html{
+    background-color: var(--gray);
+}
+
+nav {
+    justify-content: space-evenly;
+    gap: var(--sp-1);
+    width: 95vw;
+    padding: var(--sp-0-25) var(--sp-1);
+    align-items: center;
+    background-color: transparent;
+}
+
+.nav-title {
+    flex-direction: row;
+    color: var(--white);
+    transition: all linear 0.3s;
+}
+
+.title-highlight {
+    color: var(--primary-color);
+}
+.nav-title:hover {
+    flex-direction: row-reverse;
+    cursor: pointer;
+}
+
+.nav-title:first-child:hover {
+    transform: translateX(5px);
+}
+
+input {
```

Comment:
Suggestion - use a class to change it, if it is not a global style

Reply:
I've made the suggested changes, thank you!